frontend/www/js/omegaup/contest/edit.ts

```diff
@@ -25,6 +25,7 @@ OmegaUp.on('ready', () => {
     data: () => ({
       admins: payload.admins,
       details: payload.details,
+      initialTab: window.location.hash ? window.location.hash.substring(1) : '',
```

Comment:
Here you can simplify as follows:
```suggestion
      initialTab: window.location.hash?.substring(1) || '',
```
and it might be a good idea to always set the initial tab as 'problems'. What do you think?

Reply:
Then, when refreshed, would the tab not change to `problem` every time?

Reply:
Not really, only when the URL doesn't contain any hash, the initial tab will be `problems`.

Reply:
I see. Sure! That's a great idea, contestants expect to view the problems when they click on the contest.

Reply:
I realized that in the component, we are setting the default value according to each type of contest. So, just forget the previous messages in the thread.